selectors.yml

```diff
@@ -49,6 +49,9 @@ selectors:
   11155420:
     selector: 5224473277236331295
     name: "ethereum-testnet-sepolia-optimism-1"
+  1029:
```

Comment:
We need to sort the list by ChainID ascending so this should move up, please.

Reply:
just sorted!